<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmNavListViewForm</class>
 <widget class="QWidget" name="frmNavListViewForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="NavListView" name="listView">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page1">
      <widget class="QLabel" name="lab1">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>200</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>工业5G-U开发平台</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lab1_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>270</y>
         <width>281</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>国家重点研发计划 2020YFB1710900</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page2">
      <widget class="QWidget" name="widget_4" native="true">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>233</width>
         <height>43</height>
        </rect>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>90</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>传输模式选择</string>
        </property>
       </widget>
       <widget class="QComboBox" name="TranMode">
        <property name="geometry">
         <rect>
          <x>105</x>
          <y>9</y>
          <width>119</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>1</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QPushButton" name="confWrite">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>340</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>提交</string>
       </property>
      </widget>
      <widget class="QWidget" name="widget_5" native="true">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>10</y>
         <width>233</width>
         <height>43</height>
        </rect>
       </property>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>90</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>时隙选项设置</string>
        </property>
       </widget>
       <widget class="QComboBox" name="LCh0_TransType">
        <property name="geometry">
         <rect>
          <x>105</x>
          <y>9</y>
          <width>119</width>
          <height>25</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="comboBox_3">
        <property name="geometry">
         <rect>
          <x>-140</x>
          <y>10</y>
          <width>119</width>
          <height>25</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_6" native="true">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>233</width>
         <height>43</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>功率选项设置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="powerfactor">
          <property name="text">
           <string>1000</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget_7" native="true">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>40</y>
         <width>233</width>
         <height>43</height>
        </rect>
       </property>
       <widget class="QLabel" name="TranMode_2">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>90</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>天线模式选择</string>
        </property>
       </widget>
       <widget class="QComboBox" name="ue_slots">
        <property name="geometry">
         <rect>
          <x>105</x>
          <y>9</y>
          <width>119</width>
          <height>25</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_8" native="true">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>233</width>
         <height>43</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>接收频率设置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="rx_freq">
          <property name="text">
           <string>3150000000L</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget_9" native="true">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>70</y>
         <width>233</width>
         <height>43</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>发送频率设置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="tx_freq">
          <property name="text">
           <string>3950000000L</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget_10" native="true">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>233</width>
         <height>43</height>
        </rect>
       </property>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>源端  IP  设置</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="LCh0_Src_ip">
        <property name="geometry">
         <rect>
          <x>105</x>
          <y>9</y>
          <width>119</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>&quot;192.168.2.212&quot;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_11" native="true">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>100</y>
         <width>233</width>
         <height>43</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>源点端口设置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LCh0_Src_port">
          <property name="text">
           <string>8888</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget_12" native="true">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>233</width>
         <height>43</height>
        </rect>
       </property>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>89</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>目的端IP设置</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="LCh0_Dst_ip">
        <property name="geometry">
         <rect>
          <x>104</x>
          <y>9</y>
          <width>120</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>&quot;192.168.2.191&quot;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_13" native="true">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>130</y>
         <width>233</width>
         <height>43</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>目的端口设置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LCh0_Dst_port">
          <property name="text">
           <string>6680</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page3">
      <widget class="QComboBox" name="Source">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>511</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>参数源码</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>传输层源码</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>...</string>
        </property>
       </item>
      </widget>
      <widget class="QTextEdit" name="SourceEdit">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>37</y>
         <width>511</width>
         <height>431</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="writeButton">
       <property name="geometry">
        <rect>
         <x>279</x>
         <y>479</y>
         <width>241</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>更新源文件</string>
       </property>
      </widget>
      <widget class="QPushButton" name="readButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>479</y>
         <width>241</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>读取源文件</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page4">
      <widget class="QLabel" name="yclabel">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>190</y>
         <width>511</width>
         <height>458</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>3</height>
        </size>
       </property>
       <property name="toolTipDuration">
        <number>-1</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QCheckBox" name="netCheckBox">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>160</y>
         <width>85</width>
         <height>29</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777212</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>远程访问</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="iplabel">
       <property name="geometry">
        <rect>
         <x>177</x>
         <y>217</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>IP地址:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ipEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>215</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
       <property name="inputMask">
        <string>000.000.000.000</string>
       </property>
       <property name="text">
        <string>192.168.0.1</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="dklabel">
       <property name="geometry">
        <rect>
         <x>174</x>
         <y>262</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>端口号:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="dkEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>260</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>21</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavListView</class>
   <extends>QListView</extends>
   <header>navlistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
